gen/golden_test.go

```diff
@@ -42,6 +42,10 @@ var noZapFiles = map[string]struct{}{
 	"nozap": {},
 }
 
+var errOnUnknownEnumFiles = map[string]struct{}{
+	"err_on_unknown_enum_value" : {},
```

Comment:
I don't see a `internal/tests/thrift/err_on_unknown_enum_value` directory.

Reply:
Yeah should this be `enum-text-marshal-strict`?